packages/stream-tile-handler/src/tile-document-handler.ts

```diff
@@ -137,13 +137,6 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
         // TODO: Assert that the 'prev' of the commit being applied is the end of the log in 'state'
         const proof = (await context.ipfs.dag.get(commit.proof, { timeout: IPFS_GET_TIMEOUT })).value;
 
-        const supportedChains = await context.api.getSupportedChains()
```

Comment:
This isn't actually necessary and isn't really the right place to be doing this check anyway. The AnchorValidator already has a check for if the chainId changes from what the CAS has configured, this check is redundant and doesn't work when there is no CAS available